src/vcf/assembly_checker.cpp

```diff
@@ -121,7 +121,7 @@ namespace ebi
           if (rc.get()) {
               fetch_contig_on_demand = true;
           }
-
+          std::unordered_set<std::string> downloadedContigs;
```

Comment:
This should be named `downloaded_contigs` in order to follow our naming conventions.